src/types/zetkin.ts

```diff
@@ -234,34 +234,46 @@ export type { ZetkinView, ZetkinViewColumn, ZetkinViewRow };
 export interface ZetkinJourney {
   id: number;
   organization: ZetkinOrganization;
-  plural_name: string;
-  singular_name: string;
+  plural_label: string;
+  singular_label: string;
   stats: {
     closed: number;
     open: number;
   };
 }
```

Comment:
Maybe not now but a `types/journeys.ts` might be good to have soon

Reply:
Im gonna do this in my PR anyway :+1:

Reply:
That's great!